adapters/33across/33across.go

```diff
@@ -243,8 +243,8 @@ func (a *TtxAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalRequ
 
 func validateVideoParams(video *openrtb2.Video, prod string) (*openrtb2.Video, error) {
 	videoCopy := *video
-	if videoCopy.W == 0 ||
-		videoCopy.H == 0 ||
+	if (videoCopy.W == nil || *videoCopy.W == 0) ||
+		(videoCopy.H == nil || *videoCopy.H == 0) ||
```

Comment:
We would like to keep the checks for 0 since we don't expect requests with a 0x0 size.

All the changes to our 33across adapter look good!

cc: @curlyblueeagle